user/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    first_name      = models.CharField(max_length = 64, null  = False)
+    last_name       = models.CharField(max_length = 64, null  = False)
+    email           = models.EmailField(unique    = True)
+    phone           = models.CharField(max_length = 64, null  = False, unique = True)
+    password        = models.CharField(max_length = 128, null = False)
+    birth           = models.CharField(max_length = 128, null = True)
```

Comment:
```suggestion
    birthdate           = models.DateField(max_length = 128, null = True)
```
birthdate가 보다 명확한 이름이라고 추천드립니다. 날짜를 저장할 때에는 date field를 사용합니다

Reply:
이름을 변경하였고, 필드는 DateField를 사용하는 방식으로 수정했습니다.
옵션중 max_length는 삭제했고, null옵션은 그대로 사용하도록 했습니다.